sematic/resolvers/local_resolver.py

```diff
@@ -76,6 +99,11 @@ def __init__(self, rerun_from: Optional[str] = None, **kwargs):
 
         self._rerun_from_run_id = rerun_from
 
+        # this parameter needs to be resolved after the root future is enqueued
+        # and before the resolution is created
+        self._cache_namespace: Optional[str] = None
```

Comment:
```suggestion
        self._cache_namespace_str: Optional[str] = None
```

To make it more clear that this is the version after we have turned it into a string?